README.md

```diff
@@ -170,13 +168,13 @@ the Function constructor arguments passed the `IsCodeLike` check.
 
 ##### Pros
 
-- Simple host interface; Does not require passing objects to the host.
 - No TOCTOU issues - code-like object passing the check is immediately
   stringified before the host callout happens.
```

Comment:
This is not actually true anymore, since the order of operations now is not anymore
- check
- get the string
- call the host
but just
- get the string
- call the host (which will check)

I think we can still write that there are no TOCTOU issue because the check is performed including the string version of the code that will be executed.

Reply:
Given this is gonna be moved to a PR to the spec directly I don't think the readme is super important, but I will do a run through once it's in a finalised state and try to make it as clear and up to date as possible